_topic_maps/_topic_map.yml

```
@@ -1013,6 +1013,8 @@ Topics:
     File: cert-manager-operator-issuer-acme
   - Name: Configuring the egress proxy for the cert-manager Operator for Red Hat OpenShift
     File: cert-manager-operator-proxy
+  - Name: Authenticating the cert-manager Operator for Red Hat OpenShift without GCP workload identity
```

```suggestion
  - Name: Authenticating the cert-manager Operator for Red Hat OpenShift on GCP
```
PTAL at all similar references and make required change.